lib/matcher.js

```diff
@@ -1,3 +1,4 @@
+var _ = require('lodash');
 var PROPERTY_TO_FLAG_AS_STRUMMER_MATCHER = "IS-STRUMMER-MATCHER";
 
 function Matcher(opts) {
```

Comment:
Or could we support

```js
function Matcher(opts) {
  this.optional = opts && (opts.optional === true);
  this.name = opts ? opts.name : null;
}
```

I thought a nice thing about `strummer v2` was that there's nothing special about `s(...)` anymore, it's just a shortcut to compile the matcher if necessary.